qemu_v8.mk

```diff
@@ -65,7 +65,9 @@ arm-tf-clean:
 # QEMU
 ################################################################################
 qemu:
-	cd $(QEMU_PATH); ./configure --target-list=aarch64-softmmu\
+	cd $(QEMU_PATH); \
+		git submodule update --init dtc && \
```

Comment:
@jbech-linaro do you mean it will also clone submodules that are not needed? It is true and indeed it's a problem because some of them are quite big (u-boot). Still, having `make` start by downloading stuff kind of makes me sick... ;) so let's try something else. What about:

```diff
        <!-- QEMU -->
-       <project remote="qemu" path="qemu" name="qemu.git" />
+       <project remote="qemu" path="dtc" name="dtc.git" />
+       <project remote="qemu" path="qemu" name="qemu.git">
+               <linkfile src="../dtc" dest="dtc" />
+       </project>
```
?